rules.ini

```
@@ -183,6 +183,7 @@ AMD_FSR2 = (?:^|/)ffx_fsr2_api_x(?:64|86)\.dll$
 AMD_GPU_Services = (?:^|/)amd_ags_x(?:64|86)\.dll$
 AMD_TrueAudio_Next = (?:^|/)TrueAudioNext\.dll$
 Anzu = (?:^|/)anzu\.dll$
+Autodesk_FBX = (?:|/)fbxsdk
```

to also match the unity plugin
```suggestion
Autodesk_FBX = fbxsdk
```